rpc-tests/tests/contract.js

```diff
@@ -0,0 +1,50 @@
+// 1. Import the contract abi
+import compiled from './compile.js';
+
+// 2. Add the Web3 provider logic here:
+import Web3 from 'web3';
+
+// Create Web3 instance
+const web3 = new Web3('http://localhost:9933');
+
+// 5. Create get function
+export const sayMessage = async (contractAddress) => {
+    const abi = compiled.abi;
+    // 4. Create contract instance
+    const hello = new web3.eth.Contract(abi, contractAddress);
+    console.log(`Making a call to contract at address: ${contractAddress}`);
+
+    // 6. Call contract
+    const data = await hello.methods.sayMessage().call();
+
+    console.log(`The current message is: ${data}`);
+
+    return data;
+};
+
+export const setMessage = async (contractAddress, accountFrom, message) => {
```

Comment:
```suggestion
export const setMessage = async (contractAddress, accountFrom, message): Promise<void> => {
```

Reply:
Its not using typescript for now. I will move it to typescript in the next PR